includes/class-wcpdf-font-synchronizer.php

```diff
@@ -17,7 +17,7 @@ class Font_Synchronizer {
 	 *
 	 * @var string
 	 */
-	public $font_cache_filename = "dompdf_font_family_cache.php";
+	public $font_cache_filename = "installed-fonts.dist.json";
```

Comment:
I think this shouldn't be `.dist`, right? See:
https://github.com/dompdf/dompdf/blob/79573d8b8a141ec8a17312515de8740eed014fa9/src/FontMetrics.php#L36